src/components/ContentPlaceholder/ContentPlaceholder.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './ContentPlaceholder.scss';
+
+const { block, elem } = bem({
+    name: 'ContentPlaceholder',
+    classnames: styles,
+    propsToMods: []
+});
+
+const calcDuration = seconds => {
+    if (!seconds) {
+        return null;
+    }
+
+    return `${seconds}s`;
+};
+
+const calcMaskWidth = width => {
+    if (!width) {
+        return null;
+    }
+
+    return {
+        width: `${100 - Math.max(Math.min(width, 100), 0)}%`
+    };
+};
+
+const ContentPlaceholder = props => {
+    const { duration, height, width, ...rest } = props;
+
+    return (
+        <div
+            {...rest}
+            {...block(props)}
+            style={{
+                animationDuration: calcDuration(duration),
+                height
+            }}
+        >
+            &nbsp;
```

Comment:
necessary?

Reply:
Yes I added this white space on purpose, otherwise the box will not have a width / height

Reply:
cool